packages/cli-platform-android/src/commands/runAndroid/getTaskNames.ts

```diff
@@ -1,13 +1,42 @@
 import {toPascalCase} from './toPascalCase';
 import type {BuildFlags} from '../buildAndroid';
+import {getGradleTasks} from './listAndroidTasks';
+import {CLIError, logger} from '@react-native-community/cli-tools';
 
 export function getTaskNames(
   appName: string,
   mode: BuildFlags['mode'] = 'debug',
   tasks: BuildFlags['tasks'],
   taskPrefix: 'assemble' | 'install',
+  sourceDir: string,
 ): Array<string> {
-  const appTasks = tasks || [taskPrefix + toPascalCase(mode)];
+  let appTasks = tasks || [taskPrefix + toPascalCase(mode)];
+  console.log('appTasks', appTasks);
+  console.log('taskPrefixe', taskPrefix);
```

Comment:
`taskPrefixe` -> `taskPrefix` ;-)

Btw. do console logs stay or are they up for removal?

Reply:
Yeah, they're! :)